ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java

```diff
@@ -19,17 +19,17 @@
 
 public class CrosslinkCommittee {
```

Comment:
Is this the right time to rename this? Crosslinks don't exist anymore so I think this is something like `BeaconCommittee` now? Or maybe just `Committee`.  Happy to leave it for a future PR though.

Reply:
done